tests/integrationv2/providers.py

```diff
@@ -688,27 +694,27 @@ def get_send_marker(cls):
     def create_priority_str(self):
         priority_str = "NONE"
 
-        if self.options.protocol:
-            priority_str += ":+" + \
-                self.protocol_to_priority_str(self.options.protocol)
```

Comment:
In case this change isn't obvious:

The functions `*_to_priority_str` sometimes return `None` even when their argument is not `None`. This results in a type error whey you attempt to and `None` to the `priority_str`.